README.md

```diff
@@ -986,11 +990,22 @@ inference responses returned by a decoupled model. If the `decoupled` parameter
 is set to `False`, the `exec` and `async_exec` function will return a single
 response as shown in the example above.
 
-Besides, you can set the timeout via the parameter 'timeout' in microseconds
-within the constructor of `InferenceRequest`. If the request times out, the
-request will respond with an error. The default of 'timeout' is 0 which
-indicates that the request has no timeout. Example below shows how to use this
-feature:
+Additionally, starting from the 23.04 release, you have the flexibility to
+select a specific device to receive output tensors from BLS calls. This
+can be achieved by setting the optional `preferred_memory` parameter within the
+`InferenceRequest` constructor. To do this, you can create a `PreferredMemory`
+object and specify the `preferred_memory_type` as either `PreferredMemory.GPU`
+or `PreferredMemory.CPU`, as well as the `preferred_device_id` as an integer to
+indicate the memory type and device ID on which you wish to receive output
+tensors. In the event that the `preferred_memory_type` is set to
+`PreferredMemory.GPU` but the device with the specified `preferred_device_id`
+is unavailable for output allocation, the Python backend will attempt to
+allocate output tensors on other available devices and only return an error if
+none of the devices are available. If you do not specify the `preferred_memory`
+parameter, the output tensors will be allocated on the same device where the
+output tensors were received from the model to which the BLS call is made.
```

Comment:
@GuanLuo  Do you think the current design is flexible enough to support host policies if needed in the future?